src/implementations/vanilla/utilities/aria/README.md

```diff
@@ -0,0 +1,3 @@
+# ECL Utility Aria
```

Comment:
I'm not sure that these utlites are needed anymore. I don't think that they are used, and there is also no demo of it.
So I'd suggest to remove them, and if they appear to be needed later we can bring them back, with a demo

Reply:
Removed